external.bzl

```diff
@@ -691,9 +691,9 @@ def kythe_dependencies():
     maybe(
         http_archive,
         name = "com_google_protobuf",
-        sha256 = "b50be32ea806bdb948c22595ba0742c75dc2f8799865def414cf27ea5706f2b7",
-        strip_prefix = "protobuf-3.7.0",
-        urls = ["https://github.com/protocolbuffers/protobuf/archive/v3.7.0.zip"],
```

Comment:
nit, optional: Add a reminder to update to the next tagged release when available?